Dockerization finished

duskcloudxu opened the pull request:
Hi, I have finished the dockerization and updated the README, please test it according to the instruction in the README, and any suggestion on typos or expression is appreciated.

Angela-Liu commented on Dockerfile:
```diff
@@ -0,0 +1,18 @@
+RUN mkdir /banana-rails
+WORKDIR /banana-rails
+COPY . /banana-rails
+# RUN apt-get update -qq && apt-get install -y nodejs postglesql-client
```
Do you still need this comment?

duskcloudxu replied:
I removed that line, thanks!

256hz commented on entrypoint.sh:
```diff
@@ -0,0 +1,10 @@
+set -e
+
+# Remove a potentially pre-existing server.pid for Rails.
+rm -f /myapp/tmp/pids/server.pid
```
I don't think this would break if the file wasn't there, but we could consider doing it conditionally like below

`[ ! -e /myapp/tmp/pids/server.pid ] || rm -f /myapp/tmp/pids/server.pid`

duskcloudxu commented:
Hi guys I fixed the problem we met in the day by using the postgres 12 image,  tried it on windows and updated the setup instructions for windows.

There is a playground website for docker: https://labs.play-with-docker.com. Very useful to test the docker. I tested the project on it and it works fine. I guess it now works on your machine. Could you help to test it?  @Angela-Liu @256hz
